src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -382,6 +386,20 @@ ReportClientSideStats(
   }
   std::cout << "    Throughput: " << stats.infer_per_sec << " infer/sec"
             << std::endl;
+
+  if (verbose) {
+    std::stringstream client_overhead{""};
+    std::stringstream send_rate{""};
+    client_overhead << "    "
```

Comment:
If we have a warning based off of this, and we report the real request rate, do we need to print the overhead?  It will help us in debug, but will it confuse the user?

Reply:
If we are leaving it up then we will have to clearly document what it is (in our upcoming doc stories)